src/ui/features/xp-drop/components/ClaimXpBanner/ClaimXpBanner.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import DropIcon from 'jsx:src/ui/assets/drop.svg';
+import { HStack } from 'src/ui/ui-kit/HStack';
+import { GradientBorder } from 'src/ui/components/GradientBorder';
+import { UIText } from 'src/ui/ui-kit/UIText';
+import { Button } from 'src/ui/ui-kit/Button';
+import { Link } from 'react-router-dom';
+
+export function ClaimXpBanner() {
```

Comment:
This banner is not used anywhere

Reply:
yep, another leftover, actually the new for that banner is `XpDropClaimBanner` (which is used), thanks, removed ✅